includes/class-wc-gateway-ppec-admin-handler.php

```diff
@@ -153,22 +153,27 @@ public function maybe_capture_charge( $order ) {
 	 * @param int $order_id
 	 */
 	public function capture_payment( $order_id ) {
-		$order  = wc_get_order( $order_id );
-		$old_wc = version_compare( WC_VERSION, '3.0', '<' );
+		$order = wc_get_order( $order_id );
```

Comment:
Just a note on these changes. I noticed while testing that when I completed the checkout I would get the following PayPal Request and Response. 

```
WC_Gateway_PPEC_Client::_request: remote request to https://api-3t.sandbox.paypal.com/nvp with params: Array
(
    [TRANSACTIONID] => 
    [METHOD] => GetTransactionDetails
    [VERSION] => 120.0
    [USER] => ....business.example.com
    [PWD] => ...
    [SIGNATURE] => ...
)

WC_Gateway_PPEC_Client::_process_response: acknowleged response body: Array
(
    [ADDRESSOWNER] => PayPal
    [ADDRESSSTATUS] => None
    [TIMESTAMP] => 2020-03-24T02:39:16Z
    [CORRELATIONID] => f010a41f9b40e
    [ACK] => Failure
    [VERSION] => 120.0
    [BUILD] => 53802284
    [L_ERRORCODE0] => 10011
    [L_SHORTMESSAGE0] => Invalid transaction id value
    [L_LONGMESSAGE0] => A transaction id is required
    [L_SEVERITYCODE0] => Error
    [PENDINGREASON] => None
    [REASONCODE] => None
    [L_TAXABLE0] => false
)
```

I narrowed it down to this function which is trying to work out if we need to capture an authorized payment. Because there's no payment required, there's no transaction ID and therefore when we were trying to get the transaction details to determine if it needed capture we'd send an empty ID. 

These orders don't need auth and capture so when there's no transaction ID, don't continue. 

You can replicate this error on `master` by simply purchasing a free trial subscription with the latest changes from #698.